doc/detailsDefault.md

```diff
@@ -0,0 +1,47 @@
+# Details of Default File
+
+This file stores the **id** of commands, **id** and **compName** of events, and **id** and **compName** of telemetry.
+LabVIEW Component will read this file to register commands, events and telemetry in TCP Server.
+
+The **default.ini** file is in `config/` directory.
+You can follow [here](../config/default.ini) for details.
+
+## Explanation of Items
+
+### commands
+
+1. **id** : Specify the name of command that begins with `cmd_`.
+
+### events
+
+1. **compName** : Specify which component's event is being subscribed.
+2. **id** : Specify the name of event that begins with `evt_`.
```

Comment:
In the `ini` file, the evt_` is not needed. Right?

Reply:
You are right, I will remove the `evt_`.